app/forms/DistributionsIncludedFormProvider.scala

```diff
@@ -31,12 +31,12 @@ class DistributionsIncludedFormProvider @Inject() extends Mappings {
           "distributionsIncluded.error.invalid"
         ),
         "distributionsIncludedAmount" -> conditional[Int, DistributionsIncluded](
-          positiveWholeAmount(
+          wholeAmount(
             "distributionsIncludedAmount.error.required",
-            "distributionsIncludedAmount.error.outOfRange",
+            "error.outOfRange",
             "distributionsIncludedAmount.error.doNotUseDecimals",
             "distributionsIncludedAmount.error.nonNumeric"
-          ).verifying(inRange(1, 2147483647, "distributionsIncludedAmount.error.outOfRange"))
+          ).verifying(minimumValue(1, "error.lessThanOne"), maximumValue(1000000000, "error.greaterThanOneBillion"))
```

Comment:
Can we move this to a constant please - 1000000000

Reply:
Where would the constant be located? In utils?
Something like we had before?

val ONE_BILLION = 1000000000

And I assume that we would then use the constant everywhere we encounter "1000000000" value?